Loadouts/msv_ak74_emr.hpp

```diff
@@ -0,0 +1,323 @@
+  headgear[] = {"rhs_fieldcap_digi"};
+  items[] += {RADIO_LR};
+};
+class potato_msv_aplt: potato_msv_plt {}; // PLT Assistant
+class potato_msv_coy: potato_msv_plt { // COY
+  headgear[] = {"rhs_tsh4"};
+};
+class potato_msv_acoy: potato_msv_plt{}; // COY Assistant
+class potato_msv_pol: potato_msv_rifleman { // Poloitical Officer
+  vest[] = {"rhs_vest_commander"};
+  headgear[] = {"rhs_beret_mp1"};
+  backpack[] = {}; // This loadout looks stupid with a backpack
+  backpackItems[] = {BASE_MEDICAL};
+  weapons[] = {"rhs_weap_makarov_pm"};
+  magazines[] = {"rhs_mag_9x18_8_57N181S:3"};
+  items[] += {RADIO_MR,RADIO_MR}; // 2 radios
+};
+class potato_msv_AR: potato_msv_rifleman {// AR
+  weapons[] = {AR};
+  magazines[] = {AR_MAG,PISTOL_MAG,BASE_GRENADES};
+  handguns[] = {PISTOL};
+  attachments[] = {};
+};
+class potato_msv_g: Fic_Soldier_Carbine {// Grenadier
+  magazines[] += {AT_MAG};
+  launchers[] = {AT};
+};
+class potato_msv_ag: Fic_Soldier_Carbine {// Assistant Grenadier
+  magazines[] += {AT_MAG};
+};
+class potato_msv_sm: Fic_Soldier_Carbine {// Medic
+  vest[] = {"rhs_6b23_digi_medic"};
+  magazines[] = {CARBINE_MAG,MEDIC_GRENADES};
+  backpackItems[] = {MEDIC_MEDICAL};
+};
+class potato_msv_marksman: potato_msv_rifleman {// Squad Marksman
+  weapons[] = {MARKSMAN};
+  magazines[] = {MARKSMAN_MAG};
+  items[] += {"ACE_RangeCard"};
+  attachments[] = {"rhs_acc_pso1m2"};
+};
+class Fic_Spotter: potato_msv_rifleman {
+  linkedItems[] += {RANGE_FINDER};
+};
+class potato_msv_mmgg: potato_msv_AR {// MMG
+  backpack[] = {"B_Kitbag_sgg"};
+  weapons[] = {MMG};
+  magazines[] = {MMG_MAG,PISTOL_MAG,BASE_GRENADES};
+  attachments[] = {};
+};
+class potato_msv_mmgag: Fic_Spotter {// MMG Spotter/Ammo Bearer
+  backpack[] = {"B_Kitbag_sgg"};
+  backpackItems[] += {MMG_MAG};
+};
+class potato_msv_matg: Fic_Soldier_Carbine {// MAT Gunner
+  backpack[] = {"rhs_rpg_empty"};
+  backpackItems[] = {};
+  magazines[] += {MAT_MAG};
+  items[] += {BASE_MEDICAL};
+  launchers[] = {MAT};
+  secondaryAttachments[] = {MAT_OPTIC};
+};
+class potato_msv_matag: Fic_Spotter {// MAT Spotter/Ammo Bearer
+  backpack[] = {"rhs_rpg_empty"};
+  backpackItems[] = {};
+  magazines[] += {MAT_MAG2};
+  items[] += {BASE_MEDICAL};
+};
+class potato_msv_msamg: Fic_soldier_Carbine {// SAM Gunner
+  SAM_GEAR("B_Carryall_oli", SAM_MAG)
+  launchers[] = {SAM};
+};
+class potato_msv_msamag: Fic_Spotter {// SAM Spotter/Ammo Bearer
+  SAM_GEAR("B_Carryall_oli", SAM_MAG)
+};
+class potato_msv_msaml: potato_msv_msamag {// SAM Leader
+  items[] += {LEADER_TOOLS};
+  linkedItems[] += {LEADER_LINKED,RANGE_FINDER};
+};
+class potato_msv_mtrl: Fic_Soldier_Carbine { // Mortar Leader
+  items[] += {LEADER_TOOLS};
+  linkedItems[] += {LEADER_LINKED,RANGE_FINDER};
+};
+class potato_msv_mtrg: Fic_Soldier_Carbine {// Mortar Gunner
+  MORTAR_GEAR("O_Mortar_01_weapon_F")
+};
+class potato_msv_mtrag: Fic_Spotter {// Assistant Mortar
+  MORTAR_GEAR("O_Mortar_01_support_F")
+};
```

Comment:
Since this is intended to only be run on CO-OPs now do they need one? How does everyone feel about them not getting them? The marines won't have MR/LR on their weapons teams but the weapons for USMC is more integrated.